extensions/smallrye-reactive-messaging-kafka/deployment/src/test/java/io/quarkus/smallrye/reactivemessaging/kafka/deployment/testing/PriceConverter.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.smallrye.reactivemessaging.kafka.deployment.testing;
+
+//
+//import io.smallrye.reactive.messaging.annotations.Broadcast;
+//import org.eclipse.microprofile.reactive.messaging.Incoming;
+//import org.eclipse.microprofile.reactive.messaging.Outgoing;
+//
+//import javax.enterprise.context.ApplicationScoped;
+//
+//@ApplicationScoped
+public class PriceConverter {
+    //    private static final double CONVERSION_RATE = 0.88;
+    //
+    //    @Incoming("prices")
+    //    @Outgoing("processed-prices")
+    //    @Broadcast
+    //    public double process(int priceInUsd) {
+    //        return priceInUsd * CONVERSION_RATE;
+    //    }
```

Comment:
Is this normal all this code is commented out?

Reply:
Yes, it simulates starting with an empty class and then adding the code. In the dev mode test I just remove the comments.